src/js/components/Video/Video.js

```diff
@@ -455,53 +469,56 @@ const Video = forwardRef(
     }
 
     return (
-      <StyledVideoContainer
-        ref={containerRef}
-        {...mouseEventListeners}
-        alignSelf={alignSelf}
-        gridArea={gridArea}
-        margin={margin}
-        style={style}
-      >
-        <StyledVideo
-          {...rest}
-          ref={videoRef}
-          onDurationChange={(event) => {
-            const video = videoRef.current;
-            setDuration(video.duration);
-            setPercentagePlayed((video.currentTime / video.duration) * 100);
-            if (onDurationChange) onDurationChange(event);
-          }}
-          onEnded={(event) => {
-            setPlaying(false);
-            if (onEnded) onEnded(event);
-          }}
-          onPause={(event) => {
-            setPlaying(false);
-            if (onPause) onPause(event);
-          }}
-          onPlay={(event) => {
-            setPlaying(true);
-            setHasPlayed(true);
-            if (onPlay) onPlay(event);
-          }}
-          onTimeUpdate={(event) => {
-            const video = videoRef.current;
-            setCurrentTime(video.currentTime);
-            setPercentagePlayed((video.currentTime / video.duration) * 100);
-            if (onTimeUpdate) onTimeUpdate(event);
-          }}
-          onVolumeChange={(event) => {
-            setVolume(videoRef.current.volume);
-            if (onVolumeChange) onVolumeChange(event);
-          }}
-          autoPlay={autoPlay || false}
-          loop={loop || false}
+      <Keyboard onRight={seekForward} onLeft={seekBackward}>
```

Comment:
I noted an issue regarding the lack of ability to access the controls after video play. Would it be appropriate to add an `onTab` here with a callback that sets `setInteracting(true)`?

Reply:
Yes I think that is a good idea. I will add that